app/components/forum/forum-post-new.hbs

```diff
@@ -1,5 +1,5 @@
 <p class='card-text'>
-  <MdEditor @content={{content}} @textareaId='newForumPost' />
+  <MdEditor @content={{this.content}} @textareaId='newForumPost' />
```

Comment:
note that this no longer refers to the `content` argument, but rather to the `content` value that comes from the controller. In this situation, that `content` property on the controller comes in the form of a getter.
If we wanted to still refer to the `content` argument, we would need to write `@content`
(this is an effect of glimmer components)
I hope this demonstrates how glimmer components clarify what we are referencing

Reply:
I also find that this nicely demonstrates how glimmer components will clarify code in general